src/Test/Tasty/Plutarch/Property.hs

```diff
@@ -102,6 +105,38 @@ peqProperty expected gen shr comp =
                 Left e -> unexpectedError e
                 Right s' -> sameAsExpected s'
 
+{- | 'alwaysFailProperty' universally checks if given computation fails.
+ This runs the given script with generated input; it makes sure that
+ script fails by crashes or errors.  Since output does not need to be
+ checked, `PEq` instance is not required.
```

Comment:
The last sentence isn't really necessary IMHO.

Reply:
yes agreed